alphanets/alpaca/manifest.yaml

```diff
@@ -0,0 +1,33 @@
+name: alpaca
+type: alphanet
+l1:
+  name: sepolia
+  chain_id: 11155111
+l2:
+  deployment:
+    op-deployer:
+      version: v0.0.9
+    l1-contracts:
+      locator: https://storage.googleapis.com/oplabs-contract-artifacts/artifacts-v1-c193a1863182092bc6cb723e523e8313a0f4b6e9c9636513927f1db74c047c15.tar.gz
```

Comment:
These locators should be tags. L1 is on 1.8.0-rc.4, and L2 is on the 1.7 beta tag still.